src/contexts/Providers.jsx

```diff
@@ -0,0 +1,13 @@
+import { SellProvider } from "./SellContext";
+import { AuthProvider } from "./AuthContext";
+
+// TODO move the sell provider to just the /sell routes
+export function Providers({ children }) {
```

Comment:
Note with this change, the auth context will be available to all components (which seems to make sense to me but shout if you think otherwise). The sell provider should be limited to components that just need it but this should be fine for now

Reply:
Should we put public_navigation bar component under the sell provider?

Reply:
Include another light blue navigation bar under the sell provider.